src-tests/test-rpc.lisp

```diff
@@ -4,6 +4,20 @@
 
 (in-package #:rpcq-tests)
 
+;; TODO FIXME XXX The below relies on some funky/hacky/unsightly code
```

Comment:
Perhaps it could be desirable to have this also as an issue on Github.

Reply:
I consider fixing this to be part of #75, but I've added a note there now to make sure this doesn't fall through the cracks.

https://github.com/rigetti/rpcq/issues/75#issuecomment-574375441